portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase

```diff
@@ -375,6 +375,139 @@ definition {
 		}
 	}
 
+	@description = "Bug ID: LRAC-8093 | Automation ID: LRAC-11533 | Test Summary: Able to clear property data"
+	@priority = "4"
+	test ClearPropertyData {
+		property analytics.cloud.upstream = "false";
+		property test.name.skip.portal.instance = "Properties#ClearPropertyData";
+
+		var acProperty = "Clear Property";
+
+		task ("Launch AC and go to settings and add a new property") {
+			ACUtils.launchAC();
+
+			ACNavigation.goToSettings();
+
+			ACSettings.goToProperties();
+
+			ACProperties.addNewProperty(propertyName = "${acProperty}");
```

Comment:
I have the same question that manu asked too. I think you could use the property that is created at connection time. Whenever you want to access the value of the property you are using in the test, just use the variable:

`${assignedPropertyName}`

What do you think?

Reply:
Yea I'll just do that instead. I initially was thinking of doing this in an empty workspace so I wanted the variables for the new workspace and property above. But since I changed it it's probably no longer needed. 
cc: @manuferreira